Blish HUD/Controls/TextInputBase.cs

```diff
@@ -583,14 +575,14 @@ public abstract class TextInputBase : Control {
         protected virtual void HandlePaste() {
             ClipboardUtil.WindowsClipboardService.GetTextAsync()
                          .ContinueWith((clipboardTask) => {
-                              if (!clipboardTask.IsFaulted) {
-                                  if (!string.IsNullOrEmpty(clipboardTask.Result)) {
-                                      Paste(clipboardTask.Result);
-                                  }
-                              } else {
+                             if (!clipboardTask.IsFaulted) {
```

Comment:
It might be better to use
```csharp
(!clipboardResult.IsCompleted)
```
instead in a ContinueWith since IsCompleted covers 
Canceled, Faulted and RanToCompletion according to [docs](https://docs.microsoft.com/de-de/dotnet/api/system.threading.tasks.task.iscompleted?redirectedfrom=MSDN&view=net-5.0#System_Threading_Tasks_Task_IsCompleted).